addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/handler/YamahaReceiverHandler.java

```diff
@@ -185,15 +195,18 @@ private void updateReceiverState() {
         }
         lastRefreshInMS = System.currentTimeMillis();
 
+        boolean includeNetradioStation = isLinked(YamahaReceiverBindingConstants.CHANNEL_NETRADIO_STATION);
+
         try {
-            state.updateState();
```

Comment:
See comment below. I favor a solution where you call a separate method like updateNetRadioMenu() if includeNetradioStation is true.